src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -85,6 +92,7 @@ public static Person[] getSamplePersons() {
                        new DateOfBirth("1990-01-01"),
                        new Gender("M"),
                        getTagSet("colleagues"),
+                       new Priority(DEFAULT_PRIORITY),
```

Comment:
Yeah the DEFAULT_PROPERTY is being repeated, will be resolved with default constructor mentioned above

Reply:
Amended